### PR TITLE
[Auto] Localization - Translated Strings

### PR DESCRIPTION
**Pull request by csigs**

### CODE REVIEW

File: Extension/i18n/esn/package.i18n.json [hunk 1]
```diff
@@ -167,16 +167,16 @@
 	"c_cpp.configuration.suggestSnippets.description": "Si se establece en true, el servidor de lenguaje proporciona los fragmentos de código.",
 	"c_cpp.configuration.enhancedColorization.description": "Si se habilita esta opción, el código se colorea de acuerdo con IntelliSense. Esta configuración solo se aplica si intelliSenseEngine se establece en \"Predeterminado\".",
 	"c_cpp.configuration.codeFolding.description": "Si está habilitada, el servidor de lenguaje proporciona intervalos de plegado de código.",
-	"c_cpp.configuration.vcpkg.enabled.markdownDescription": "Habilita los servicios de integración para el [administrador de dependencias de vcpkg](https://aka.ms/vcpkg/).",
+	"c_cpp.configuration.vcpkg.enabled.markdownDescription": "Habilita los servicios de integración para el [administrador de dependencias de vcpkg] (https://aka.ms/vcpkg/).",
```
Comment (Colengms) [inline Extension/i18n/esn/package.i18n.json]: I opened a feature request against VS Code that might help us address this issue: https://github.com/microsoft/vscode/issues/125774

**Comment by sean-mcmanus** [comment]
Don't check this in yet -- it regresses the fixes we made -- the translator teams needs to fix the issues on their side.

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]
Regresses stuff, don't check in yet...

**Review by sean-mcmanus — APPROVED** [review]

**Comment by sean-mcmanus** [comment]
@Colengms Does this seem okay now?

File: Extension/i18n/chs/package.i18n.json [hunk 2]
```diff
@@ -167,7 +167,7 @@
 	"c_cpp.configuration.suggestSnippets.description": "如果为 true，则由语言服务器提供片段。",
 	"c_cpp.configuration.enhancedColorization.description": "如果启用，则根据 IntelliSense 对代码设定颜色。此设置仅在 intelliSenseEngine 设置为“默认”时适用。",
 	"c_cpp.configuration.codeFolding.description": "如果启用，则由语言服务器提供代码折叠范围。",
-	"c_cpp.configuration.vcpkg.enabled.markdownDescription": "为 [vcpkg 依赖关系管理器] 启用集成服务(https://aka.ms/vcpkg/)。",
+	"c_cpp.configuration.vcpkg.enabled.markdownDescription": "为 [vcpkg 依赖关系管理器](https://aka.ms/vcpkg/)启用集成服务。",
```
Comment (bobbrow) [inline Extension/i18n/chs/package.i18n.json:170]: Should there be a space after the link?

Comment (bobbrow) [reply Extension/i18n/chs/package.i18n.json:170]: I guess not

Comment (sean-mcmanus) [reply Extension/i18n/chs/package.i18n.json:170]: Not sure -- the change was from the translator.

Comment (sean-mcmanus) [reply Extension/i18n/chs/package.i18n.json:170]: I just fixed the space that broke the hyperlink.

**Pull request merged**